src/plugins/lightbox/lightbox.js

```diff
@@ -315,7 +315,7 @@ var componentName = "wb-lbx",
 
 		$wrap.on( "keydown", function( e ) {
 			if ( e.which === 9 ) {
-				var tabbable = $wrap.find( ".mfp-container :tabbable:visible" ),
+				var tabbable = $wrap.find( ".mfp-container a[href]:visible, .mfp-container button:visible, .mfp-container input:visible, .mfp-container textarea:visible, .mfp-container select:visible, .mfp-container details>summary:visible, .mfp-container [tabindex]:not([tabindex='-1']:visible)" ),
```

Comment:
Check if we need to update the jQuery selector or not

* [tabbable](https://github.com/wet-boew/wet-boew/blob/master/src/core/helpers.js#L1547)